packs/equipment-effects/effect-spellguard-shield.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Spellguard Shield",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spellguard Shield]</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spellguard Shield]</p>\n<p>You gain a +2 circumstance bonus to saving throws against spells that target you (as well as to AC), and you can Shield Block spells that target you if you have that action.</p>"
```

Comment:
I think the ac bonus RE can be removed, as it's already granted by raising the shield. That said, this effect is now defunct as the item itself contains the necessary REs.

Reply:
I'll remove the effect from the item and remove the ac bonus.

Reply:
Nevermind - the shield doesn't have the effect.